rosie/chamber_of_deputies/tests/test_monthly_subquota_limit_classifier.py

```diff
@@ -18,15 +18,15 @@ def setUp(self):
 
     def test_predict_false_when_not_in_date_range(self):
         assert_array_equal(np.repeat(False, 10),
-                           self.prediction[[0, 1, 9, 10, 18, 19, 27, 28, 36, 37]])
+                           self.prediction[[0, 1, 18, 19, 27, 28, 36, 37, 45, 46]])
```

Comment:
Quick note here: car renting before 07/2017 is not really out of range as it falls into the previous period (04/2015 - 06/2017)